static/app/components/sidebar/index.tsx

```diff
@@ -355,10 +360,10 @@ function Sidebar() {
   );
 
   const resources = hasOrganization && (
-    <Feature key="resource" features="spans-first-ui">
+    <Feature key="assets" features="spans-first-ui">
```

Comment:
```suggestion
    <Feature key="resource" features="spans-first-ui">
```

You can leave it as-is, so it matches with the definition of `ModuleName` enum